skycoin/skywire
# improve survey logic

Did you run `make format && make check`? Yes

Fixes #_

 Changes:	
- add new flag `--raw-survey` to skywire-visor for generate survey decrypted
- add routine job (each 24h) to re-generate survey

How to test this PR:
- run visor `skywire-visor -c skywire-config.json --raw-survey` and check node-info.json for raw/decrypted data there
